src/state/queries/notifications/unread.tsx

```diff
@@ -46,61 +63,88 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
   // listen for broadcasts
   React.useEffect(() => {
     const listener = ({data}: MessageEvent) => {
-      lastSyncRef.current = new Date()
+      cacheRef.current = {
+        sessDid: currentAccount?.did || '',
+        syncedAt: new Date(),
+        data: undefined,
+      }
```

Comment:
What do you think about invalidating the query here too if the last sync is behind in this tab?

Reply:
Might trigger more work than we need, but worth considering. Let's see how things feel